src/state/apis/nft/nftApi.ts

```diff
@@ -53,6 +53,25 @@ type NftState = {
   }
 }
 
+const NFT_NAME_BLACKLIST = [
+  'voucher',
+  'airdrop',
+  'giveaway',
+  'promo',
+  'airdrop',
+  'rewards',
+  'ticket',
+  'winner',
+  '$',
+  'pirategirls',
+  ' USDC',
+  'calim cryptopunk',
+  'coupon',
+]
+const nftNameBlacklistRegex = new RegExp(
+  NFT_NAME_BLACKLIST.map(term => term.replace(/[.*+?^${}()|[\]\\]/g, '\\$&')).join('|'),
```

Comment:
Regexes are pain, but tl;dr is this escapes special characters just in case

Reply:
would be worth commenting this